README.md

```diff
@@ -1,11 +1,35 @@
 
+**Robotoff Documentation:** <https://openfoodfacts.github.io/robotoff>
+
 # Current models
 * [tf-universal-logo-detector-1.0](https://github.com/openfoodfacts/robotoff-models/releases/tag/tf-universal-logo-detector-1.0) related to [robotoff-ann](https://github.com/openfoodfacts/robotoff-ann), some additional info in [this openfoodfacts-ai directory](https://github.com/openfoodfacts/openfoodfacts-ai/tree/develop/logo-ann)
 * [tf-nutrition-table-1.0](https://github.com/openfoodfacts/robotoff-models/releases/tag/tf-nutrition-table-1.0), some additional info in [this openfoodfacts-ai directory](https://github.com/openfoodfacts/openfoodfacts-ai/tree/develop/nutrition-table-extraction)
 * [tf-nutriscore-1.0](https://github.com/openfoodfacts/robotoff-models/releases/tag/tf-nutriscore-1.0), some additional info in [nutriscore_tensorflow](https://github.com/openfoodfacts/nutriscore_tensorflow)
 * [keras-category-classifier-xx-2.0](https://github.com/openfoodfacts/robotoff-models/releases/tag/keras-category-classifier-xx-2.0) related to [off-category-classification](https://github.com/openfoodfacts/off-category-classification)
 * [category-predictor-xgfood-emlyon-1.0](https://github.com/openfoodfacts/robotoff-models/releases/tag/category-predictor-xgfood-emlyon-1.0) related to [off-category-classification](https://github.com/openfoodfacts/off-category-classification), some additional info in [this openfoodfacts-ai directory](https://github.com/openfoodfacts/openfoodfacts-ai/tree/develop/ai-emlyon)
 * [category-predictor-ocr-lewagon-1.0](https://github.com/openfoodfacts/robotoff-models/releases/tag/category-predictor-ocr-lewagon-1.0) related to [off-category-classification](https://github.com/openfoodfacts/off-category-classification)
+
+## Weekly meetings
+- We e-meet Mondays at 17:00 Paris Time (16:00 London Time, 21:30 IST, 08:00 AM PT)
+- ![Google Meet](https://img.shields.io/badge/Google%20Meet-00897B?logo=google-meet&logoColor=white) Video call link: https://meet.google.com/qvv-grzm-gzb
+- Join by phone: https://tel.meet/qvv-grzm-gzb?pin=9965177492770
+- Add the Event to your Calendar by [adding the Open Food Facts community calendar to your calendar](https://wiki.openfoodfacts.org/Events)
+- [Weekly Agenda](https://drive.google.com/open?id=1RUfmWHjtFVaBcvQ17YfXu6FW6oRFWg-2lncljG0giKI): please add the Agenda items as early as you can. Make sure to check the Agenda items in advance of the meeting, so that we have the most informed discussions possible. 
+- The meeting will handle Agenda items first, and if time permits, collaborative bug triage.
+- We strive to timebox the core of the meeting (decision making) to 30 minutes, with an optional free discussion/live debugging afterwards.
+- We take comprehensive notes in the Weekly Agenda of agenda item discussions and of decisions taken.
```

Comment:
We take comprehensive notes in the Weekly Agenda document of the items discussed and decisions made.